### DISCUSSION TITLE
Nextcloud AIO -  503 when accessing login after install

### DISCUSSION THREAD

**Discussion by Gamechiefx** [nextcloud/all-in-one#1594] [Questions] [answered]
Hello,

I have NPM installed as the reverse proxy. I have all docker containers running however i am unable to access on port 443 but can get the AIO login on 8080

please help

**Comment by szaimen** [comment, marked as answer]
Hi, did you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md?
